### ISSUE TITLE
question: python3 (venv3.sh) support for certbot-dns-rfc2136 plugin?

### ISSUE THREAD

**Issue by hal869** [certbot/certbot#6224]
I'm switching over to dns-01 challenge usage.

I run a local DNS server, with dynamic update support, so plan to use the "certbot-dns-rfc2136" plugin.

Following

  https://certbot.eff.org/docs/contributing.html,

, DL'ing latest certbot git source,
```
	git log | head
		commit 595e77eccbd58131db7a3e07ad9385f436119dc3
		Author: Trinopoty Biswas <trinopoty@hotmail.com>
		Date:   Mon Jul 16 21:10:16 2018 +0530

		    Fixed linode API settings page URL (#6208)

		commit 72daec4346e3e43eed1ba4f42d5ab6e3346e8798
		Author: Brad Warren <bmw@users.noreply.github.com>
		Date:   Mon Jul 16 08:38:50 2018 -0700
```
I notice that the tools/venv.sh script *does* include the plugin,
```
	cat venv.sh
		...
		./tools/_venv_common.sh \
		  -e acme[dev] \
		  ...
		  -e certbot-dns-rfc2136 \
		...
```
but, the python3 tools/venv3.sh script does NOT
```
	grep rfc venv3.sh
		(empty)
```
Is that a simple oversight? Or is this plugin not supported/recommended for use with python3?

If python3 is supported, but not via venv(3).sh, what's the recommended plugin install method for python3 env?

cc @bmw

**Comment by bmw** [comment]
It's just an oversight. Would you like to make a PR to add the plugin to the script?

**Comment by hal869** [comment]
@bmw 

> make a PR to add the plugin to the script?

https://github.com/certbot/certbot/pull/6226

**Comment by bmw** [comment]
Fixed in #6226.
